nautobot/extras/datasources/git.py

```diff
@@ -1050,6 +1050,15 @@ def delete_git_graphql_queries(repository_record, job_result, preserve=None):
                 job_result.log(error_msg, level_choice=LogLevelChoices.LOG_ERROR, grouping="graphql queries")
 
 
+def refresh_git_data_compliance_rules(repository_record, job_result, delete=False):  # pylint: disable=W0613
+    """Callback for repo refresh."""
+    from nautobot.nautobot_data_validation_engine.custom_validators import get_classes_from_git_repo
```

Comment:
Can we rename this method to `get_data_compliance_classes_from_git_repo` to be less generic?

Reply:
Sorry, I think I was unclear. I was referring to renaming `get_classes_from_git_repo`, not renaming `refresh_git_data_compliance_rules`!